### DISCUSSION TITLE
Permission denied: '/opt/airflow/dag_generated_configs

### DISCUSSION THREAD

**Discussion by joserodriguezc** [open-metadata/OpenMetadata#8829] [Q&A]
Hello:

I would like help to resolve this error: [[Errno 13] Permission denied: '/opt/airflow/dag_generated_configs/s3_demo_metadata_uwq8rvfs.json'] "}

I get it when I want to configure a metadata ingestion from S3

It's a Docker Deployment
thanks!

**Comment by yerkogallardo** [comment]
You must to to change the permissions on volume mount point with chmod 775, with that you can work without problems.

Cheers!
